.github/workflows/checkrefs.yml

```diff
@@ -0,0 +1,22 @@
+name: checkrefs
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  checkrefs:
+    runs-on: bullseye
+    steps:
+    - uses: actions/checkout@v3
+    - run: |
+        FILES=$(git log -1 -p -- maps -- simulation | grep +++ | cut -d '/' -f 2-| sed -e 's|dev/null||g')
```

Comment:
@LangLangBart Well, I'm not too sure about this. It doesn't seem to work as intended. Let me know if you have suggestions about that. Otherwise, we might live with it for a while and see how it works in the wild.